.github/workflows/unit_tests.yml

```
@@ -14,12 +14,15 @@ jobs:
   unit_tests:
     name: unit tests
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.10", "3.11", "3.12", "3.13"]
```

Add 3.9 to match #18:
```suggestion
        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
```